Phase08/Phase08/Phase08/DisjunctionFilter.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using SQLHandler;
+
+namespace Phase08
+{
+    public class DisjunctionFilter : IFilter
+    {
+        private readonly InvertedIndexContext _invertedIndexContext;
+
+        public DisjunctionFilter(InvertedIndexContext invertedIndexContext)
+        {
+            _invertedIndexContext = invertedIndexContext;
+        }
+
+        public ISet<string> Filter(ISet<string> signQueries)
+        {
+            var disjunctionFiltered = new HashSet<string>();
+            
+            return signQueries.Aggregate(disjunctionFiltered, (current, query) =>
+                current.Union(_invertedIndexContext.GetDocumentsContainQuery(query)).
+                    ToHashSet());
```

Comment:
afarin
inja ke GetDocumentsContainQuery to class khoadesh gozashti kar khobie
faghat hatman wrapper bekeshin rosh va ye interface bezarin barash ke be ye class depend nabashe